docs/reference/tls-acme.md

```diff
@@ -263,3 +263,14 @@ dns namedotcom {
 }
 ```
 
+- rfc2136 (non-default)
+
+```
+dns rfc2136 {
+    KeyName "..."
+    Key "..."
+    KeyAlg "..."
+    Server "..."
```

Comment:
```suggestion
    key_name "..."
    key "..."
    key_alg "..."
    server "..."
```

Reply:
Though providing examples would be useful for `key_alg` and `server`. Especially `key_alg`.

Reply:
Good points, I added examples, it should be clearer now.